packages/asc-ui/src/components/InfoCard/InfoCard.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react'
+import { Close } from '@datapunt/asc-assets'
+import InfoCardStyle, {
+  Props as StyleProps,
+  CloseButton,
+  Levels,
+  CloseButtonWrapper,
+} from './InfoCardStyle'
+import Heading from '../Heading'
+import CompactThemeProvider from '../../theme/CompactThemeProvider'
+import Paragraph from '../Paragraph'
+import Link from '../Link'
+import { ButtonVariants } from '../Button/ButtonStyle'
+
+type Props = {
+  heading?: string
+  onClose?: Function
+  content?: string
+  linkProps?: {
+    href: string
+    title: string
+  }
+} & StyleProps
+
+const InfoCard: React.FC<Props & React.HTMLAttributes<HTMLElement>> = ({
```

Comment:
There are some concerns here in regards to accessibility. We should probably allow this component to be focussed using keyboard navigation and add information for screen readers on how to read the content inside of the component such as `aria-live="polite"`.

The current order of elements also causes the 'close' button to come into focus before anything else which might be confusing for people using screen readers.